#Системный флаг
int8 flag 